layer/utils/runtime_utils.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any
```

Comment:
Will this file be used in the executors? If not, perhaps I'd move it under `layer/decorators/` folder.

Reply:
The intention is to be used by an executor, yes.